native/App.tsx

```diff
@@ -94,19 +92,17 @@ const App = () => {
   }, [])
 
   // Sends the saved user to the web client to make the log in process faster
-  useEffect(() => {
-    if (listeningToNative && fbUser) {
-      // We use a timeout because sometimes the auth persistence manager is still undefined on the client side
-      // Seems my iPhone 12 mini can regularly handle a shorter timeout
-      setTimeout(() => {
-        communicateWithWebview('nativeFbUser', fbUser)
-      }, 100)
-      // My older android phone needs a bit longer
-      setTimeout(() => {
-        communicateWithWebview('nativeFbUser', fbUser)
-      }, 250)
-    }
-  }, [listeningToNative, fbUser])
+  const sendWebviewAuthInfo = (user: FirebaseUser) => {
```

Comment:
Now that this is called directly, if the ` auth.onAuthStateChanged` is actually used for anything, we won't get updates from the useEffect. If people are left signed out we'll probably want to add this back in.